tests/tck/conftest.py

```diff
@@ -980,3 +980,19 @@ def switch_to_new_session(conn_pool, user, password, class_fixture_variables, ex
     sess = conn_pool.get_session(user, password)
     class_fixture_variables["sessions"].append(sess)
     exec_ctx["current_session"] = sess
+
+@when(parse('login with user "{user}"'))
+def login_without_password(conn_pool, user):
+    sess = None
+    try:
+        sess = conn_pool.get_session(user, '')
```

Comment:
why not to reset the current_session in exec_ctx?

Reply:
Just verify logging, avoid affecting other tests.

Reply:
if you don't change the current session, how do you make sure the next test cases use the current login user?

Reply:
Next test will use origin session.

Reply:
if so, does this step just test whether the user exists?  why not to use `show users` to test it? if you want to switch user and test role, plz consider to switch the current session.

Reply:
This will be logged in by this user.

Reply:
Just test login feature? what to do after the user login?  I don't find where to use the login user?

Reply:
Just test login.

Reply:
why not to test this feature in role cases ? such as:

```
When executing query:
  """
  create user username...
  grant user some rule ..
  """
Then executing successfully
When switch to new session with user "{user}" and password "{password}"
Then executing successfully
When executing query:
  """
   test role of user
  """
Then ...
```

it's not necessary to create a new step for this feature and we have the related step for switching user:

```
switch to new session with user "{user}" and password "{password}"
```

Reply:
Don't want to change current session.